blimflix/client/app/favorite-movies/favorite-movies.route.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Separate configuration file for modules
+ *   https://github.com/johnpapa/angular-styleguide#style-y129
+ */
+(function () {
+  'use strict';
+
+  angular.module('blimFlix.favoriteMovies').config(config);
+
+  /* @ngInject */
+  function config($routeProvider) {
+    /**
+     * Always declare controllers with their routes
+     *   https://github.com/johnpapa/angular-styleguide#style-y038
+     */
+    $routeProvider.when('/favorite-movies', {
```

Comment:
@Fuksia, and you will need to change this to be:

`$routeProvider.when('/favorites', {`